modules/linux

```diff
@@ -22,7 +25,11 @@ linux_kconfig := $(or $(CONFIG_LINUX_CONFIG),config/linux.config)
 linux_dir := $(linux_base_dir)/$(notdir $(basename $(linux_kconfig)))
 
 linux_tar := linux-$(linux_version).tar.xz
+ifeq "$(CONFIG_LINUX_VERSION)" "5.4.69"
```

Comment:
probably need a cleaner check than this, especially if we need to support multiple 5.x kernels in the future. open to suggestions.

Reply:
@https://stackoverflow.com/questions/2741708/makefile-contains-string ?
Tweaked to begin search at the beginning of literal?

Reply:
@tlaurion ignoring readability,
`linux_url := https://cdn.kernel.org/pub/linux/kernel/v$(basename $(basename $(CONFIG_LINUX_VERSION))).x/$(linux_tar)` should work for all 2-3-digit kernel versions, if I'm reading the makefile docs correctly